airbyte-webapp-e2e-tests/cypress/e2e/connection/connectionStreamDetails.cy.ts

```diff
@@ -0,0 +1,141 @@
+import {
+  getConnectionCreateRequest,
+  getPostgresCreateDestinationBody,
+  getPostgresCreateSourceBody,
+  requestCreateConnection,
+  requestCreateDestination,
+  requestCreateSource,
+  requestDeleteConnection,
+  requestDeleteDestination,
+  requestDeleteSource,
+  requestSourceDiscoverSchema,
+  requestWorkspaceId,
+} from "commands/api";
+import { Connection, Destination, DestinationSyncMode, Source, SourceSyncMode } from "commands/api/types";
+import { appendRandomString } from "commands/common";
+import { runDbQuery } from "commands/db/db";
+import {
+  createAccountsTableQuery,
+  createUserCarsTableQuery,
+  dropAccountsTableQuery,
+  dropUserCarsTableQuery,
+  getCreateUsersTableQuery,
+  getDropUsersTableQuery,
+} from "commands/db/queries";
+import { initialSetupCompleted } from "commands/workspaces";
+import * as connectionPage from "pages/connection/connectionPageObject";
+import { StreamDetailsPageObject } from "pages/connection/streamDetailsPageObject";
+import { NewStreamsTablePageObject } from "pages/connection/streamsTablePageObject/NewStreamsTablePageObject";
+
+const dropTables = () => {
+  runDbQuery(
+    getDropUsersTableQuery("users"),
+    getDropUsersTableQuery("users2"),
+    dropAccountsTableQuery,
+    dropUserCarsTableQuery
+  );
+};
+
+describe.skip("Connection - Stream details", () => {
```

Comment:
Should this be skipped?

Reply:
Yes, because the new table is disabled. To run this locally, you'd have to remove this temporarily.